pkg/provider/aci.go

```diff
@@ -492,11 +498,19 @@ func (p *ACIProvider) GetPod(ctx context.Context, namespace, name string) (*v1.P
 		return nil, err
 	}
 
-	err = validation.ValidateContainerGroup(cg)
+	retry.OnError(retry.DefaultBackoff,
+		func(err error) bool {
+			return true
+		}, func() error {
+			err = validation.ValidateContainerGroup(ctx, cg)
```

Comment:
Why do you need to retry without refreshing cg by calling GetContainerGroupInfo again?